lib/cloud_controller/diego/lifecycles/cnb_lifecycle.rb

```diff
@@ -0,0 +1,56 @@
+require 'cloud_controller/diego/lifecycles/buildpack_info'
+require 'cloud_controller/diego/lifecycles/buildpack_lifecycle_data_validator'
+require 'fetchers/buildpack_lifecycle_fetcher'
+
+module VCAP::CloudController
+  class CNBLifecycle
```

Comment:
Is it possible to reuse `BuildpackLifecycle`?

Reply:
done